cq/apps.py

```diff
@@ -21,7 +22,12 @@ def requeue_tasks(*args, **kwargs):
     lock = 'RETRY_QUEUED_TASKS'
     with cache.lock(lock, timeout=2):
         # Find all Queued tasks and set them to Retry, since they get stuck after a reboot
-        Task.objects.filter(status=Task.STATUS_QUEUED).update(status=Task.STATUS_RETRY)
+        try:
+            tasks = Task.objects.filter(status=Task.STATUS_QUEUED).update(status=Task.STATUS_RETRY)
+            logger.info('Requeued {} tasks'.format(tasks))
+        except ProgrammingError:
+            logger.info('Not requeuing tasks')
```

Comment:
Add a comment here (or extend the logging) to explain why we'd be falling into this error.